operations/patch.go

```diff
@@ -0,0 +1,157 @@
+package operations
```

Comment:
compile errors: 
operations/patch.go:67:4: undefined: comm
operations/patch.go:76:59: undefined: comm
operations/patch.go:143:43: not enough arguments in call to params.validatePatchCommand
	have (*ClientSettings, *legacyClient)
	want (context.Context, *ClientSettings, *legacyClient, client.Communicator)
operations/patch.go:143:43: too many errors